borg/archiver.py

```diff
@@ -462,6 +463,15 @@ def do_prune(self, args):
             stats.print_('Deleted data:', cache)
         return self.exit_code
 
+    def do_convert(self, args):
+        """convert a repository from attic to borg"""
+        repo = AtticRepositoryConverter(args.repository.path, create=False)
+        try:
+            repo.convert(args.dry_run)
+        except NotImplementedError as e:
+            print("warning: %s" % e)
```

Comment:
just as a reminder to change this to logging.warning (or .error) after we have reasonable logging in place.